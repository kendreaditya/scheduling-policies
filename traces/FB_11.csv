1,0,100
2,50,57
3,100,50
4,100,13
5,110,37
6,130,57
7,150,30
8,150,0
9,150,0
10,150,0
11,160,10
12,165,10
13,170,57
14,175,55
15,180,53
16,195,51
17,200,49
18,205,47
19,210,45
20,215,43
