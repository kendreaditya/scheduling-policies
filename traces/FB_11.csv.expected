1,764
2,721
3,684
4,289
5,570
6,721
7,493
8,150
9,150
10,150
11,260
12,260
13,721
14,713
15,703
16,691
17,678
18,664
19,648
20,630
